organizations/organizations.shacl.ttl

```diff
@@ -430,8 +451,12 @@ haOrg:OrganizationalUnitShape a sh:NodeShape ;
     sh:message "schema:email is missing or the datatype is no xsd:string."@en ;
   ],
   [
+    a sh:PropertyShape ;
     sh:path haOrg:isAccountManagerOf ;
-    sh:class org:Organization ;
+    sh:or (
```

Comment:
Dit klopt niet. Een instantie van`haOrg:ContentPartner` en `haOrg:EducationalPartner` moet een accountmanager hebben, maar alle andere instanties van `org:Organization` mogen een accountmanager hebben. Maw, een persoon kan accountmanager zijn van alle instanties van `org:Organization`

Reply:
Dank je voor de verheldering ik heb deze teruggezet. Ik zie wel dat een persoon maar van maximaal 1 organisatie account manager mag zijn. Klopt dit of mag een Persoon ook van meerdere organisaties accountmanager zijn?

Reply:
oei nee, een persoon moet accountmanager kunnen zijn van meerdere organisaties (een organisatie mag wel maar 1 accountmanager hebben)

Reply:
De aanpassing is doorgevoerd. Een persoon kan van meer dan 1 organisatie accountmanager zijn. Maar een organisatie kan maximaal 1 accountmanager hebben.